playwright-e2e/pages/singular/home/home-page.ts

```diff
@@ -0,0 +1,58 @@
+import { expect, Locator, Page } from '@playwright/test';
+import { SingularPage } from 'pages/singular/singular-page';
+import { fillEmailPassword } from 'authentication/auth-singular';
+import { HomePageInterface } from 'pages/page-interface';
+
+/**
+ * Landing page, unauthenticated.
+ */
+export default class HomePage extends SingularPage implements HomePageInterface {
+  private readonly logo: Locator;
+  private readonly title: Locator;
+  private readonly description: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.title = this.page.locator('h1.header-title');
+    this.logo = this.page.locator('a.header__logo img');
+    this.description = this.page.locator('.header-description');
+  }
+
+  async waitForReady(): Promise<void> {
+    // Add additional waits here
+    await this.page.waitForFunction(() => document.title === 'Project Singular');
+    await expect(this.title).toContainText('Discoveries in single ventricle heart defects are on the horizon.');
+  }
+
+  /**
+   * Returns "Sign me up!" button locator
+   */
+  getSignMeUpButton(): Locator {
+    return this.page.locator('header').locator('button', { hasText: 'Sign me up!' });
+  }
+
+  /**
+   * Click "Sign me up!" button.
+   * @param opts
+   */
+  async signUp(opts: { waitForNav?: boolean } = {}): Promise<void> {
+    const { waitForNav = true } = opts;
+    await this.clickAndWaitForNav(this.getSignMeUpButton(), { waitForNav });
+  }
+
+  /**
+   * Log into Singular application
+   * @param opts
+   */
+  async logIn(
+    opts: { email?: string | undefined; password?: string | undefined; waitForNavigation?: boolean } = {}
```

Comment:
I believe we can skip `undefined` here because optional parameter already means that it can be undefined.
so just 
`  opts: { email?: string; password?: string; waitForNavigation?: boolean } = {}`
would be enough.